arches/app/media/js/views/components/datatypes/file-list.js

```diff
@@ -19,6 +20,7 @@ define([
             params.config.imagesOnly(val);
         });
 
+        params.config.imagesOnly.subscribe((val) => self.imagesOnly(val));
```

Comment:
It looks like we can directly use the config.imagesOnly observable from params?
`this.imagesOnly = params.config.imagesOnly;`